html_css/js/index.js

```diff
@@ -1,37 +1,38 @@
 import preferDropDown from './drop-down-menu.js';
 import initMobileMenu from './mobile-menu.js';
-import paginator from './paginator.js'
-// import paginatorData from './data/paginator-data.js'
-import paginatorData from './data/paginator-data-big.js'
+// import paginatorData from './data/paginator-data.js';
+import paginatorData from './data/paginator-data-big.js';
+import App from './app.js';
+// import '../css/style.css'
```

Comment:
Do you need it?

Reply:
// import '../css/style.css'
used when WebPack minifiing files